setup.py

```diff
@@ -35,6 +35,9 @@
         "sphinxcontrib-openapi>=0.8.0",
         "sphinxcontrib-redoc>=1.5.1",
     ],
+    "htcondor": [
+        "htcondor==9.0.17",
```

Comment:
FYI the latest version of HTCondor ships prebuilt wheels for ARM64 on PyPI: https://github.com/htcondor/htcondor/releases/tag/v23.1.0

Maybe we can (later on) investigate whether we can build older versions of the HTCondor's Python package?

Reply:
LXPLUS9 contains HTCondor 10.3.1 so we might be upgrading soon indeed....